### DISCUSSION TITLE
Boost at 10% steps like the physical tank

### DISCUSSION THREAD

**Discussion by flipside101** [tomasmcguinness/homeassistant-mixergy#65] [General]
Howdy, not sure if its here or in HA i should ask but  im looking to make a simple boost card in HA akin to the one on the tank that will boost in 10% steps. Im had  ago setting helpers and automation to do it but nothing seems to work 

Any suggestions ?

**Comment by tomasmcguinness** [comment]
Are you still looking for help with this?

**Reply by flipside101** [reply]
great timing, i had stopped thinking about it whilst dealt with sensor issues and then trying to get the replacement tank fitted.

In short yes
